### DISCUSSION TITLE
useidentitycolumn RESOLVIDO

### DISCUSSION THREAD

**Discussion by sabrinahelena** [balta-io/forum#43] [Tira Dúvidas] [answered]
![image](https://user-images.githubusercontent.com/101990175/194712028-911b8e6e-3bd5-4889-b5d5-26d6b965ede0.png)

Eu não consigo adicionar o useidentitycolumn, alguém sabe me dizer o pq? 

GENTE JA FOI RESOLVIDO, NÃO CONSIGO APAGAR ESSA DISCUSSAO.
Estava usando rider e aí passei pro vs e deu certo.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, Sabrina, como vai?
Acredito que esteja faltando importar algum namespace:
```csharp
using System;
using System.Collections.Generic;
using Blog.Models;
using Microsoft.EntityFrameworkCore;
using Microsoft.EntityFrameworkCore.Metadata.Builders;
```

Código Fonte:
https://github.com/balta-io/2808/blob/main/src/Modulo%203/Data/Mappings
